src/controllers/OtherReasonController.ts

```diff
@@ -32,7 +32,7 @@ export class OtherReasonController extends SafeNavigationBaseController<OtherRea
     }
 
     protected prepareViewModelFromAppeal(appeal: Appeal): Record<string, any> & OtherReason {
```

Comment:
could update return types.

Reply:
When does it return `Record<string, any>`?

Reply:
The [recordkeystype](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype) for how has been implemented through all the repo is just telling us that the object has index signature of type string. The important part is the second object type, the first I would say is just a reinforce type